app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -98,6 +99,12 @@ export default class PostAttachmentOpenGraph extends PureComponent {
             ogImage = data.images.find((i) => i.url === imageUrl || i.secure_url === imageUrl);
         }
 
+        // Fallback when the ogImage does not have dimensions but there is a metaImage defined
+        const metaImages = imagesMetadata ? Object.values(imagesMetadata) : null;
+        if ((!ogImage?.width || !ogImage?.height) && metaImages?.length) {
+            ogImage = metaImages[0];
```

Comment:
not really no, `imageUrl` will remain the same, the thing here is that for the type of deployment that we have on community, the image url in the `images` metadata differs from the one in the `opengraph` metadata, thus is causing the image to fetch dimensions instead of using the ones in the metadata.

with this change we are ensuring that if the `images` metadata contains dimensions we use those.